src/routes.js

```diff
@@ -44,4 +45,16 @@ router.post('/coffeegrams', async ctx => {
   ctx.redirect(`/users/${ctx.state.currentUser.username}`);
 });
 
+router.post('/users/:username/bio', async ctx => {
```

Comment:
A `POST` endpoint in RESTful convention can be used to modify any field! You just have to add support for new fields in this route.